nesta/core/batchables/gtr/collect_gtr/run.py

```diff
@@ -54,9 +55,10 @@ def run():
     objs = []
     for table_name, rows in data.items():
         _class = get_class_by_tablename(Base, f"gtr_{table_name}")
-        # Remove any fields that aren't in the ORM
-        cleaned_rows = [{k:v for k, v in row.items() if k in _class.__dict__}
-                        for row in rows]
+        # Remove any fields that aren't in the ORM and set NULL as default
```

Comment:
The new `insert` method in `orm_utils` doesn't allow missing fields, so instead need to loop over fields in the table, rather than fields in the input data